Path-Check/covid-safe-paths
# [master 1.0.x] Copy and app flow updates for 1.0.2

Signed-off-by: Patrick Erichsen <patrick.a.erichsen@gmail.com>



#### Description:

This is a branch off of master that includes additional copy and app flow updates requested by the legal team for 1.0.2 release. Aside from misc language string updates, the following changes are included (screenshots below):

- Removes `Onboarding4.js`
- Removes the License WebView and replaces it with a link to http://covidsafepaths.org/privacy
- Updates the wording for ExposureScreen to "Coming Soon"
- Only display the mayo clinic button in the case of `StateEnum.NO_CONTACT` to prevent overlap with the CTA button on smaller screens

#### Screenshots:

<img width="391" alt="Screen Shot 2020-05-14 at 11 20 27 AM" src="https://user-images.githubusercontent.com/20157849/81959879-bd9d5980-95d5-11ea-8155-88b2d2e6bcf3.png">
<img width="392" alt="Screen Shot 2020-05-14 at 11 20 44 AM" src="https://user-images.githubusercontent.com/20157849/81959880-bece8680-95d5-11ea-8390-e0382921ec53.png">
<img width="389" alt="Screen Shot 2020-05-14 at 12 22 33 PM" src="https://user-images.githubusercontent.com/20157849/81965432-a9f5f100-95dd-11ea-8bd1-23dc0b5ec98b.png">
<img width="392" alt="Screen Shot 2020-05-14 at 12 22 45 PM" src="https://user-images.githubusercontent.com/20157849/81965445-acf0e180-95dd-11ea-975b-8715dbb85da6.png">
<img width="390" alt="Screen Shot 2020-05-14 at 1 52 24 PM" src="https://user-images.githubusercontent.com/20157849/81973729-255d9f80-95ea-11ea-873d-74c6d1b83812.png">


#### How to test:

- Verify language string changes
- Visit Legal page and confirm the new view looks satisfactory
